### ISSUE TITLE
Support InternVL2‑26B

### ISSUE THREAD

**Issue by zhly0** [huofushuo/SID#1]
您好，
看论文中的效果很好，在InternVL2‑26B中进行尝试，不知道需要怎么修改？
感谢！

**Comment by huofushuo** [comment]
Thanks for your attention. As for InternVL, you mainly need to modify the modeling strategy like modeling_llama.py--def forward() and coresponding decoding strategy like vcd_sample.py --def sample(). The corresponding intermediate variables also need to be passed.

**Comment by zhly0** [comment]
Thank you for the reply.
I am new to MLLM, could you give more specific details on which file to modify and how to modify it.
Thank you.

**Comment by huofushuo** [comment]
The corresponding intermediate variables of contrastive logits in InternVL need to be passed. InternVL forward function and decoding strategy need to be revised correspondingly.
